src/stores/app/index.js

```diff
@@ -210,7 +197,7 @@ export default class AppStore extends Store {
         await this.fetch(this.appId);
       } else {
         this.hideModal();
-        this.updateMeunApps = true;
+        // this.updateMeunApps = true;
```

Comment:
The next line code: await this.fetchAll(); add parameter menuApps, because delete app mybe change the menu apps